services/uploads/src/deps/clamAV/clamAV.test.ts

```diff
@@ -0,0 +1,80 @@
+import path from 'path'
+import { NewTestS3UploadsClient } from '../s3'
+import { NewClamAV, parseInfectedFiles } from './clamAV'
+import { rm, mkdtemp } from 'fs/promises'
+
+describe('clamAV', () => {
+    it('parses clamscan output correctly', () => {
+        const tests = [
+            {
+                output: `/Users/macrae/ccode/mc-review/managed-care-review/services/uploads/src/clamAV/testData/badList.csv: Win.Test.EICAR_HDB-1 FOUND
```

Comment:
I know these paths don't matter for this parsing test, but do we want to make the path string a little more generic than your local checkout? Just thinking of a junior dev 5 years from now wondering what this path is and who `macrae` is 😂